docs/index.md

```diff
@@ -132,11 +182,51 @@ $ docker-machine ip
 192.168.99.100
 ```
 
-Now you can manage as many local VMs running Docker as you please- just run
-`docker-machine create` again.
+For instance, you can try running a webserver ([nginx](https://nginx.org)) in a
+container with the following command:
+
+```
+$ docker run -d -p 8000:80 nginx
+```
+
+When the image is finished pulling, you can hit the server at port 8000 on the
+IP address given to you by `docker-machine ip`.  For instance:
+
+```
+$ curl $(docker-machine ip dev):8000
```

Comment:
Wanted to cover accessing a port deliberately on a machine since this seems to throw off some users.